api/Event.json

```diff
@@ -0,0 +1,1356 @@
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "isTrusted": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event/isTrusted",
+          "support": {
+            "chrome": {
+              "version_added": "46",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
+            },
+            "chrome_android": {
+              "version_added": "46",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false,
+              "notes": "In Internet Explorer, all events are trusted except those that are created with the <code>createEvent()</code> method."
+            },
+            "opera": {
+              "version_added": "33",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
+            },
+            "opera_android": {
+              "version_added": "33",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "46",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "originalTarget": {
```

Comment:
Again I'm not seeing any compat data table here.

Reply:
It’s a Mozilla-only extension that’s required by XBL and will be removed by [bug 662335](https://bugzil.la/662335 "662335 - Remove support for event.originalTarget and event.explicitOriginalTarget on DOM events").